packages/front-end/services/UserContext.tsx

```diff
@@ -157,6 +168,10 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
     mutate: refreshOrganization,
   } = useApi<OrgSettingsResponse>(isAuthenticated ? `/organization` : null);
 
+  const { data: teamsData, mutate: refreshTeams } = useApi<TeamsResponse>(
+    isAuthenticated ? `/teams` : null
+  );
```

Comment:
This doesn't feel like it needs to be a separate API call.  We should just add teams to the `GET /organization` response instead.